src/query/sql/src/planner/binder/copy_into_location.rs

```diff
@@ -109,6 +109,15 @@ impl Binder {
                     .await
             }
         }?;
+        println!("qyery pln: {:?}", &query);
```

Comment:
need add `git diff | grep print` in git commit hook


```
cd .git/hooks
 touch pre-commit
 chmod +x pre-commit 
```

```
  #!/bin/bash

   # Check for 'print' in the diff
   if git diff --cached | grep -q 'print'; then
       echo "Error: Commit contains 'print' statements."
       exit 1  # Exit with a non-zero status to prevent the commit
   fi
```